### PR TITLE
feat: add props on Loader to force user to leave screen

### PR DESCRIPTION
**Pull request by hel-axelor**
RM#73991

### CODE REVIEW

**Comment by lme-axelor** [comment]
Waiting for #367

File: packages/core/src/components/templates/Loader/LoaderPopup.tsx [hunk 1]
```diff
@@ -61,6 +63,7 @@ const LoaderPopup = ({
   const {processItem, loading} = useLoaderListner(
     {
       disabled,
+      autoLeave,
```
Comment (lme-axelor) [inline packages/core/src/components/templates/Loader/LoaderPopup.tsx:66]: I'm not sure, it's useful to add it to the process, you are not using it on the provider. 
What you did in the timeout handler should be sufficient.

File: packages/core/src/components/templates/Loader/types.ts [hunk 2]
```diff
@@ -36,6 +36,7 @@ export enum ProcessStatus {
 
 export type ProcessOption = {
   disabled: boolean;
+  autoLeave: boolean;
```
Comment (lme-axelor) [inline packages/core/src/components/templates/Loader/types.ts:39]: It's not useful here too

File: packages/core/src/components/templates/Loader/LoaderPopup.tsx (hunk 1 above)
Comment (hel-axelor) [reply packages/core/src/components/templates/Loader/LoaderPopup.tsx:66]: I added it as process item props in order to save it as history of options related to process.

**Pull request merged**